components/serializer.rst

```diff
@@ -300,21 +292,6 @@ for each format:
 
     $classMetadataFactory = new ClassMetadataFactory(new XmlFileLoader('/path/to/your/definition.xml'));
 
-.. versionadded:: 6.4
-
-    The
-    :class:`Symfony\\Component\\Serializer\\Mapping\\Loader\\AttributeLoader`
-    was introduced in Symfony 6.4. Prior to this, the
-    :class:`Symfony\\Component\\Serializer\\Mapping\\Loader\\AnnotationLoader`
-    must be used.
-
-.. deprecated:: 6.4
-
-    Reading annotations in PHP files is deprecated since Symfony 6.4.
-    Also, the
-    :class:`Symfony\\Component\\Serializer\\Mapping\\Loader\\AnnotationLoader`
-    was deprecated in Symfony 6.4.
-
 .. _component-serializer-attributes-groups-annotations:
```

Comment:
```suggestion
```

Reply:
Do we have to? I think @javiereguiluz said he wanted to avoid deleting refs at some point? I remember him saying that, but it's been a while, so I could be wrong :smile:

Reply:
I think it's fine in 7.0, even better would be to add a redirect here:
https://github.com/symfony/symfony-docs/blob/7.0/_build/redirection_map

Reply:
I mean, remove it and add an entry in the redirection_map

Reply:
First time touching this file, hope I updated it correctly :+1: